### PR TITLE
check branch name for duplicates

### PR DESCRIPTION
**Pull request by HotThoughts**
- check branch name for duplicates

### CODE REVIEW

**Comment by HotThoughts** [comment]
Is the error in Lint the chart related to #133?

**Comment by tczekajlo** [comment]
> Is the error in Lint the chart related to #133?

Yes, it's related.

File: .github/workflows/update_rasa_x_version.yml [hunk 1]
```diff
@@ -46,6 +46,17 @@ jobs:
             echo "CREATE_UPDATE_PR=true" >> $GITHUB_ENV
           fi
 
+      - name: Check if duplicated branch exists 🔍
+        run: |
+          # find duplicates
+          BRANCH_NAME=$(curl https://api.github.com/repos/$GITHUB_REPOSITORY/branches | jq '.[].name' | grep -m 1 "bump-rasa-x-version-to-${LATEST_RASA_X_MINOR}")
```
Comment (tczekajlo) [inline .github/workflows/update_rasa_x_version.yml:52]: You don't have to use GH API, I think the `git branch` command should be enough

```
# get all remote branches
git fetch origin

BRANCH_NAME=$(git branch | grep -m 1 "bump-rasa-x-version-to-${LATEST_RASA_X_MINOR}")
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]

**Review by tczekajlo — APPROVED** [review]
LGTM 🍎

**Pull request merged**